game.po

```diff
@@ -34,82 +34,82 @@ msgstr "="
 #v1.9.17
 msgctxt "constraint_platforms"
 msgid "Platforms:"
-msgstr "Andenes:"
+msgstr "Gleise:"
 
 #v1.9.17
 msgctxt "constraint_platforms_demote"
 msgid "Demote"
-msgstr "Descender"
+msgstr "Zurückstufen"
 
 #v1.9.17
 msgctxt "constraint_platforms_promote"
 msgid "Promote"
-msgstr "Ascender"
+msgstr "Befördern"
 
 #v1.9.17
 msgctxt "dep_grp_destination"
 msgid "Destination"
-msgstr "Destino"
+msgstr "Ziel"
 
 #v1.9.17
 msgctxt "dep_grp_line"
 msgid "Line"
-msgstr "Línea"
+msgstr "Linie"
 
 #v1.9.17
 msgctxt "dep_grp_next_stop"
 msgid "Next stop"
-msgstr "Siguiente parada"
+msgstr "Nächster Halt"
 
 #v1.9.17
 msgctxt "dep_grp_train"
 msgid "Train"
-msgstr "Tren"
+msgstr "Zug"
 
 #v1.9.17
 msgctxt "dep_late"
 msgid "Max late"
-msgstr "Retraso max"
+msgstr "Maximale verspätung"
 
 #v1.9.17
 msgctxt "dep_sched"
 msgid "Scheduled"
-msgstr "Por horario"
+msgstr "Geplant"
 
 #v1.9.17
 msgctxt "destinations"
 msgid "Destinations"
-msgstr "Destinos"
+msgstr "Ziele"
 
 #v1.9.17
 msgctxt "editor_sel_only_buildings"
 msgid "Keep only buildings in the selection"
-msgstr "Mantener solo los edificios en la selección"
+msgstr "Nur Gebäude in der Auswahl behalten"
 
 #v1.9.17
 msgctxt "editor_sel_only_child_tracks"
 msgid "Keep only parallel child tracks in the selection"
-msgstr "Mantener solo las vías paralelas hijas en la selección"
+msgstr "Nur parallele Kindergleise in der Auswahl behalten"
```

Comment:
Maybe we should use something like "untergeordnete Gleise" instead of "Kindergleise"

Reply:
yeah. I couldn't think of anything better than Kinder at that point. 
That is why you always should have different ppl looking over what you have done, to verify if it makes sense or not what you have written.

Reply:
Resolved with 0cea2856a245fe98db0b586f1e5d4f299ac7771d